src/adapters/MongoStorageAdapter.js

```diff
@@ -113,9 +132,47 @@ const MongoStorageAdapter = function (uuid) {
 			cb(err, { status: "failed" });
 		});
 	};
-};
 
+	/**
+	 * Get the prefix used to filter keys for particular topics and key prefixes.
```

Comment:
```suggestion
	 * Get the prefix used to filter keys with an optional topic and keyPrefix.
	 * Note that if topic is not provided, keyPrefix is also ignored.
```
needs other suggestions below